react/features/base/dialog/components/native/PageReloadDialog.tsx

```diff
@@ -176,7 +180,8 @@ class PageReloadDialog extends Component<IPageReloadDialogProps, IPageReloadDial
                 descriptionKey = { `${t(message, { seconds: timeLeft })}` }
                 onCancel = { this._onCancel }
                 onSubmit = { this._onReloadNow }
-                title = { title } />
+                title = { title }
+                visible = { timeLeft >= 1 } />
```

Comment:
@saghul Can you confirm if we want to hide the dialog even if connection is not restored?

Reply:
Dispatch `hideDialog` instead of adding a visible flag, so it's unmounted.